wallets/provider-station/src/index.ts

```diff
@@ -75,6 +77,9 @@ export const subscribe: Subscribe = ({
       }
     },
   });
+  return () => {
```

Comment:
Do we need these? if nothing returns, it means undefined which a defined behavior in our cleanup mechanism.

Reply:
There are some other places same as here, this comment apply on them as well.

Reply:
fixed.